app/experimental_appconfig.go

```diff
@@ -102,26 +72,21 @@ func setCustomModules(app *RegenApp, interfaceRegistry types.InterfaceRegistry)
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
-	app.UpgradeKeeper.SetUpgradeHandler("Mau", func(ctx sdk.Context, plan upgradetypes.Plan) {
-		// no-op handler, does nothing
+	app.UpgradeKeeper.SetUpgradeHandler("Mau", func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		return vm, nil
```

Comment:
Should we have the original migration here instead of dummy one? I remember @aleem1314 had worked on this

Reply:
Oh I just found that it is added in stable config. Then we should remove it from here.

Also why are we integrating these new modules in stable and experimental configs?

Reply:
If we are adding something to both configs, we should directly add it to `app.go`